proto/stride/stakeibc/lsm.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package stride.stakeibc;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v8/x/stakeibc/types";
+
+enum LSMDepositStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  TRANSFER_IN_PROGRESS = 0;
+  TRANSFER_FAILED = 1;
```

Comment:
If transfer failed, it could be for temporary reasons so we would try again eventually or once it fails to transfer is it done forever?  If the former, what is the trigger for trying to resend all transfer failed tokens?  If we immediately retry then we don't need this state, if we retry at a predictable set time in the future, do we need to track how many times this token has failed to transfer and give up if the error is persistent?

Reply:
There's no expected failure scenario that I'm aware of. So a failure would likely indicate that something unpredictable and very wrong happened - at which point we would just resolve it in an upgrade